programs/governance-rewards/src/instructions/escrow/transfer.rs

```diff
@@ -0,0 +1,52 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Token, TokenAccount};
+use spl_governance::tools::spl_token;
+
+#[derive(Accounts)]
+pub struct TransferFromEscrow<'info> {
```

Comment:
I don't get the escrow functionality. The user can take tokens out of the escrow account via this instruction at any time. So how is it different from sending directly to a token account that has the user as owner?

It would make sense to me if the user could point to some authority they don't control (like a dao), and then that authority may (or may not) trigger a transfer from the escrow account to a user-owned token account. The escrow authority would have no other available actions.

Reply:
Yeah, I wasn't clear on this. But that seems like a good implementation.

Reply:
Implemented